calcom/cal.com
# add invite link to Zapier setup page

## What does this PR do?

Adds invite link to Zapier setup page + small fix because Toaster wasn't shown

<img width="1437" alt="Screenshot 2022-05-09 at 09 34 47" src="https://user-images.githubusercontent.com/30310907/167362196-87cb740e-f078-46d4-bd6e-799dd508448b.png">

## How should this be tested?

1. Go to Zapier setup page
2. Click on invite link 
3. Create new Zap and search for Cal.dev app
4. Choose trigger and test if it works as expected (invite link when running it local is for the URL cal.dev, so if you want to test zapier triggers you need to do that with cal.dev)
